src/lib/y2storage/blk_device.rb

```diff
@@ -366,6 +383,20 @@ def multipath
       children.find { |dev| dev.is?(:multipath) }
     end
 
+    # Bcache device defined on top of the device
```

Comment:
NP: What about rephrase it as: "Bcache in which the device is used as backing device."